test/fixtures/react.tsx

```diff
@@ -1,7 +1,10 @@
-import { SeamProvider } from '@seamapi/react'
+import 'isomorphic-fetch'
+
 import { render } from '@testing-library/react'
 import type { PropsWithChildren } from 'react'
 
+import { SeamProvider } from 'lib/SeamProvider.js'
```

Comment:
Did something break that required a different import?

Reply:
do we prefer @seamapi/react within the test suite? Seems like a good rule